assets/js_Library/eventHandlers.js

```diff
@@ -80,6 +80,36 @@ let grabPreviousSibling = (className, event) => {
   return sibling
 }
 
+//ALEX's CODE//
+
+// ======issue #14 - Current clocked in should be visible======
+
+/**
+ * currentJob
+ *
+ *@description  Allows the user to see what job they are currently "clocked in" to on the 'Current Job' display.
+                The user can also "clock out", thus removing the job name from the 'Current Job' display.
+ *
+ *
+ *@THOUGHTS     I would like to make it so that when a job is clocked in the other jobs 'clock in' buttons will be disabled 
+                until the current job has been clocked out. Or I guess from a freelancer viewpoint, can you be clocked into
+                multiple jobs at a time? Also, do you want to display the total time spent on the clocked in job? 
+ */
+
+let currentJob =
```

Comment:
I think this function should go in the component. There are already event listeners in the `listen()` function in the `jobsComponent` that are waiting for a clock in / clock out click. What we could do is when the clock in button is press, we just grab the name and add it to the side bar. When the clock out button is pressed we just remove the name. 

This will help to keep things organized. The clocked in job is ties to that clock in button, so we should have it's code there.